libmachine/provision/suse.go

```diff
@@ -205,3 +182,34 @@ func (provisioner *SUSEProvisioner) Provision(swarmOptions swarm.Options, authOp
 	err = provisioner.Service("docker", serviceaction.Enable)
 	return err
 }
+
+func (provisioner *SUSEProvisioner) configureFirewall() error {
+	tcpPorts := "22 80 443 2376 2379 2380 6443 9099 9796 10250 10254 30000:32767"
+	udpPorts := "8472 30000:32767"
+	cmds := make([]string, 0)
```

Comment:
nit: use nil slice declaration?
```suggestion
	var cmds []string
```

Reply:
good catch! updating in next commit.